debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlGeometry.java

```diff
@@ -5,28 +5,34 @@
  */
 package io.debezium.connector.mysql;
 
-import java.util.Arrays;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-import mil.nga.wkb.geom.Point;
-import mil.nga.wkb.io.ByteReader;
-import mil.nga.wkb.io.WkbGeometryReader;
 
 /**
- * A parser API for MySQL Geometry types, it uses geopackage-wkb-java as a base for parsing Well-Known Binary
+ * A parser API for MySQL Geometry types
  *
  * @author Omar Al-Safi
+ * @author Robert Coup
  */
 public class MySqlGeometry {
+    protected static final Logger LOGGER = LoggerFactory.getLogger(MySqlGeometry.class);
 
     private final byte[] wkb;
+    private final int srid;
+    private double[] coords;
```

Comment:
Could this be made final (I'd prefer this over any lazy calculation if feasible).